tests/test_ocs_basic_install.py

```diff
@@ -34,7 +35,8 @@ def run(**kwargs):
     log.info("Generating install-config")
     # TODO: determine better place to create cluster directories - (log dir?)
     cluster_dir_parent = "/tmp"
-    base_name = test_data.get('cluster-name', 'ocs-ci-cluster')
+    default_cluster_name = f"ocs-ci-{getuser()}"
```

Comment:
@vasukulkarni can you please rebase and probably move this to defaults as it was moved there in #27 , thanks